source/agora/consensus/EnrollmentManager.d

```diff
@@ -241,12 +241,16 @@ public class EnrollmentManager
         assumeSafeAppend(enrolls);
 
         static Enrollment[] pool_enrolls;
-        this.enroll_pool.getEnrollments(pool_enrolls);
+        this.enroll_pool.getEnrollments(pool_enrolls, height);
```

Comment:
Hmm shouldn't this be `height + 1`?

Reply:
Because the `height` passed here is the already externalized block height, we want to know if the enrollment is available for the next block right?

Reply:
> Because the `height` passed here is the already externalized block height, we want to know if the enrollment is available for the next block right?

The meaning of the parameter is the current block height and the function finds the enrollments that are available for the next height. But it can be modified as you suggested in order to reduce confusion.